components/style/properties/data.py

```diff
@@ -229,6 +229,11 @@ def explicitly_enabled_in_chrome(self):
     def enabled_in_content(self):
         return self.enabled_in == "content"
 
+    def base_type(self):
```

Comment:
s/base_type/specified_type?

Reply:
`specified_type` is already a `Longhand` method that can return a boxed type.